.eslintrc.json

```
@@ -4,10 +4,11 @@
     {
       "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
       "rules": {
-        "@nrwl/nx/enforce-module-boundaries": [
+        "@nx/enforce-module-boundaries": [
           "error",
           {
             "allow": [],
+            "checkDynamicDependenciesExceptions": ["@dsg/**"],
```

This may be better to ignore everything
```suggestion
            "checkDynamicDependenciesExceptions": ["**"],
```

---

checkDynamicDependenciesExceptions doesn't work with any version of `"**"`, `"./**"`, or `"/**"`